Updated the release notes to 2.2.1 (#28)
timescale/docs: timescaledb/overview/release-notes/index.md

while also empowering users with more control and flexibility to customize
behaviors to suit your needs.  Some of these API updates are **breaking changes**.

What's new in TimescaleDB 2.2.1:

Skip Scan optimization on single node and multinode, which significantly 
improves the performance of queries with DISTINCT ON. This 
optimization is not yet available for queries on distributed 
hypertables.

> This release also adds a function to create a distributed 
> restore point, which allows performing a consistent restore of a 
> multi-node cluster from a backup.

Reviewer: Nit: We are in general writing too long sentences in our documentation. I prefer terse and distinct text.

```suggestion
This release also adds support for creating a distributed 
restore point. This allows performing a consistent restore of a 
multi-node cluster from a backup.
```

> The bug fixes in this release address issues with size and stats 
> functions, high memory usage in distributed inserts, slow distributed 
> ORDER BY queries, indexes involving INCLUDE, and single chunk query 
> planning.

Reviewer: ```suggestion
The bug fixes in this release address issues in size and stats 
functions, high memory usage in distributed inserts, slow distributed 
ORDER BY queries, indexes involving INCLUDE, single chunk query 
planning, compression, and policies.
```

**PostgreSQL 11 deprecation announcement**

Timescale is working hard on our next exciting features. To make that 
possible, we require functionality that is unfortunately absent on 
PostgreSQL 11. For this reason, we will continue supporting PostgreSQL 
11 until mid-June 2021. Sooner to that time, we will announce the 

> specific version of TimescaleDB in which PostgreSQL 11 support will 
> not be included going forward.

Reviewer: Nit: Don't beat around the bush:
```suggestion
specific version of TimescaleDB in which PostgreSQL 11 support will be removed.
```
